runtime/shibuya/src/precompiles.rs

```diff
@@ -125,6 +127,12 @@ pub type ShibuyaPrecompilesSetAt<R, C> = (
         UnifiedAccountsPrecompile<R, UnifiedAccounts>,
         (CallableByContract, CallableByPrecompile),
     >,
+    PrecompileAt<
+        AddressU64<20487>,
+        DispatchLockdrop<R, DispatchFilterValidate<RuntimeCall, WhitelistedCalls>, ConstU32<8>>,
```

Comment:
I've seen this commented in the PR summary, but why would we use the same filter as for dispatch?

I'd suggest to add more limited one, just for the sake of security.